### PR TITLE
CDAP-7481 Adding available CDAP System Metrics in docs

### PR DESCRIPTION
**Pull request by gsps1**
Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB173-3
Passes Run 3.

Page of interest: http://builds.cask.co/artifact/CDAP-DQB173/shared/build-3/Docs-HTML/4.0.0-SNAPSHOT/en/reference-manual/http-restful-api/metrics.html#available-system-metrics

### CODE REVIEW

**Comment by awholegunch** [comment]
On-hold because I see some changes required in the RST and text.

**Comment by awholegunch** [comment]
Docs LGTM.

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst [hunk 1]
```diff
@@ -275,6 +275,76 @@ These metrics are available in a stream context:
    * - ``system.collect.bytes``
      - Number of bytes collected by the stream
 
+These metrics on request and response are available for system and user services:
```
Comment (poornachandra) [inline cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: How about the following?
`These metrics are available for a service context, both for a system service (like transaction service), or a user service context:`

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst [hunk 2]
```diff
@@ -275,6 +275,76 @@ These metrics are available in a stream context:
...
+   * - Request and Response Metric
+     - Description
+   * - ``system.request.received``
+     - Number of requests received for the service
+   * - ``system.response.successful``
+     - Number of successful responses sent
+   * - ``system.response.{server-error, client-error}``
+     - Number of ``server-error`` or ``client-error`` responses sent
+
+These metrics on logging are available on the application context:
```
Comment (poornachandra) [inline cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: `These metrics are available for every application context:`

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst [hunk 3]
```diff
@@ -275,6 +275,76 @@ These metrics are available in a stream context:
...
+   * - ``system.start.{short, long}.latency``
+     - Time taken (in milliseconds) to start ``short`` or ``long`` transactions
+   * - ``system.wal.append.count``
+     - Number of transaction edits added to the write-ahead log
+   * - ``system.{canCommit, commit, committed, inprogress, invalidate, abort}``
+     - Number of transactions in a specified transaction state
+   * - ``system.{canCommit, commit, committed, inprogress, invalidate, abort}.latency``
+     - Time taken (in milliseconds) to perform a specified transaction state update
+   * - ``system.{invalid, committing, committed, inprogress}.size``
+     - Number of transaction data structures
```
Comment (poornachandra) [inline cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: `Number of transactions of various types that are active`

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst [hunk 4]
```diff
@@ -275,6 +275,76 @@ These metrics are available in a stream context:
...
+.. list-table::
+   :header-rows: 1
+   :widths: 60 40
+
+   * - Application Logging Metric
+     - Description
+   * - ``system.app.log.{error, info, warn}``
+     - Number of ``error``, ``info``, or ``warn`` log messages logged by the application
+
+These metrics on logging are available for the system services, in the system component context:
```
Comment (poornachandra) [inline cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: `These metrics are available for the system services, in the system component context:`

File: cdap-docs/reference-manual/source/http-restful-api/metrics.rst (hunk 1 above)
Comment (poornachandra) [reply cdap-docs/reference-manual/source/http-restful-api/metrics.rst]: Does this also need a component?

**Review by poornachandra — COMMENTED** [review]
@gsps1 Few comments to address

**Comment by gsps1** [comment]
@poornachandra addressed comments

**Comment by poornachandra** [comment]
LGTM

**Pull request merged**